src/index.js

```diff
@@ -227,6 +232,12 @@ export default function undoable (reducer, rawConfig = {}) {
         debugEnd()
         return res
 
+      case config.clearHistoryType:
+        res = createHistory(state.present)
+        debug('clearing history', res)
```

Comment:
This is a detail, but `clearing history` sounds like it hasn't been done yet, which could be confusing when debugging. Maybe change this to `cleared history`?